packages/gcp/src/shared/validate-project-id.ts

```diff
@@ -0,0 +1,35 @@
+import { exec } from 'child_process';
+
+export const validateProjectID = (projectID: unknown) => {
+	if (typeof projectID === 'undefined') {
+		throw new TypeError(
+			`The 'project-id' argument must be provided, but is missing.`
+		);
+	}
+
+	if (typeof projectID !== 'string') {
+		throw new TypeError(
+			`The 'project-id' argument must be a string, but is ${JSON.stringify(
+				projectID
+			)}`
+		);
+	}
+
+	// TODO: Complete this with the node API instead of gcloud
+	exec(`gcloud projects describe ${projectID}`,
+		(error) => {
+			if (error !== null) {
```

Comment:
I got to this case but due the following error:

  WARNING: Could not open the configuration file: [/Users/jonathanburger/.config/gcloud/configurations/config_default].
  ERROR: (gcloud.projects.describe) You do not currently have an active account selected.

So yes it is best to kill it and use the Node SDK